inventory_management_system_api/repositories/catalogue_item.py

```diff
@@ -210,32 +210,43 @@ def insert_property_to_all_matching(
             session=session,
         )
 
-    def update_names_of_all_properties_with_id(
-        self, property_id: str, new_property_name: str, session: Optional[ClientSession] = None
+    def update_names_and_units_of_all_properties_with_id(
```

Comment:
I personally feel this might be better off as a separate function. The name implies both are updated when they are not, its conditional, equally we want to keep things separate when they do different unrelated things, though I can understand wanting to avoid duplication for this. Would be the same as in item.py.